arangod/Cluster/AgencyCache.cpp

```diff
@@ -469,3 +567,129 @@ void AgencyCache::invokeAllCallbacks() const {
   }
   invokeCallbacks(toCall);
 }
+
+
+AgencyCache::change_set_t AgencyCache::changedSince(
+  std::string const& what, consensus::index_t const& last) const {
+
+  static std::vector<std::string> const planGoodies ({
+      AgencyCommHelper::path(PLAN_ANALYZERS) + "/",
```

Comment:
isn't `PLAN_ANALYZER` already contains trailing `/`?

Reply:
correct, thank you